src/js/binaryen.js-post.js

```diff
@@ -2500,9 +2500,26 @@ function wrapModule(module, self = {}) {
       );
     });
   };
+  self['hasMemory'] = function() {
+    return Boolean(Module['_BinaryenHasMemory'](module));
+  };
+  self['getMemoryInfo'] = function() {
+    var memoryInfo = {
+      'module': UTF8ToString(Module['_BinaryenMemoryImportGetModule'](module)),
+      'base': UTF8ToString(Module['_BinaryenMemoryImportGetBase'](module)),
+    };
+    if (memoryInfo['module'] === '' && memoryInfo['base'] === '') {
```

Comment:
Imported memories still declare initial and max sizes, so perhaps this `if` is not necessary?

Reply:
Yep, you're right. I'll change that.

Reply:
I think a test may need updating after this change, see CI.